apps/api-example/src/nestapp/gateway.module.ts

```diff
@@ -0,0 +1,46 @@
+import {IntrospectAndCompose, RemoteGraphQLDataSource} from '@apollo/gateway'
+import {ApolloGatewayDriver, ApolloGatewayDriverConfig} from '@nestjs/apollo'
+import {Module} from '@nestjs/common'
+import {GraphQLModule} from '@nestjs/graphql'
+
+@Module({
+  imports: [
+    GraphQLModule.forRoot<ApolloGatewayDriverConfig>({
+      driver: ApolloGatewayDriver,
+      server: {
+        introspection: true,
```

Comment:
We should use the apolloIntrospection field from the config file. If thats easy possible if not thats fine with me too.